lib/s3routes/routes/routePUT.ts

```diff
@@ -200,14 +212,13 @@ function routePUT(request, response, api, log, statsClient) {
                 return routesUtils.responseNoBody(errors.MissingContentLength,
                     null, response, 411, log);
             }
-            if (Number.isNaN(request.parsedContentLength) ||
-            request.parsedContentLength < 0) {
+            if (Number.isNaN(parsedContentLength) || parsedContentLength < 0) {
                 return routesUtils.responseNoBody(errors.BadRequest,
                     null, response, 400, log);
             }
-            log.end().addDefaultFields({
-                contentLength: request.parsedContentLength,
-            });
+            // TODO What's happening?
```

Comment:
What is the issue here? Do we have a ticket number related to this that we can add to the comments?